config/application.rb

```diff
@@ -45,7 +45,5 @@ class Application < Rails::Application
     config.robots = OpenStruct.new(config_for(:robots))
 
     config.use_alma = ActiveModel::Type::Boolean.new.cast(ENV['USE_ALMA'])
-
-    config.alma = config_for(:alma).with_indifferent_access
```

Comment:
Is this related to this PR?

Reply:
also config/alma.yml is removed... I wonder if this needs to be rebased against the alma branch.